### PR TITLE
Remove trailing newline in pluralized strings

### PR DESCRIPTION
**Pull request by sergafts**
When YAML files are parsed, pluralized strings span across multiple
lines. In this case, we shouldn't add a new line after each string
during compilation since the template already includes the required
newline.

Problem and/or solution
-----------------------

How to test
-----------

Reviewer checklist
------------------

Code:
* [ ] Change is covered by unit-tests
* [ ] Code is well documented, well styled and is following [best practices](https://tem.transifex.com)
* [ ] Performance issues have been taken under consideration
* [ ] Errors and other edge-cases are handled properly

PR:
* [ ] Problem and/or solution are well-explained
* [ ] Commits have been squashed so that each one has a clear purpose
* [ ] Commits have a proper commit message [according to TEM](https://tem.transifex.com/github-guide.html#working-on-a-feature)

### CODE REVIEW

File: openformats/tests/formats/yamlinternationalization/files/1_en_exported.yml [hunk 1]
```diff
@@ -15,11 +14,10 @@ en:
     one: 'this is a test'
     other: "this is a double quote test"
 
-
   pluralized_string_with_extra_keys:
+    zero: Zero
```
Comment (kbairak) [inline openformats/tests/formats/yamlinternationalization/files/1_en_exported.yml:18]: How is this discrepancy explained?

**Review by kbairak — APPROVED** [review]
Looks solid. Just the one question.

File: openformats/tests/formats/yamlinternationalization/files/1_en_exported.yml (hunk 1 above)
Comment (sergafts) [reply openformats/tests/formats/yamlinternationalization/files/1_en_exported.yml]: I was curious as well. I saw it in the input file and added it as well but the test defines plural rules as one, many. So zero is removed.

**Review by kbairak — APPROVED** [review]
